docs/data/base/components/trap-focus/trap-focus.md

```diff
@@ -84,3 +84,10 @@ When auto focus is disabled—as in the demo below—the component only traps th
 The following demo uses the [`Portal`](/base/react-portal/) component to render a subset of the `TrapFocus` children into a new "subtree" outside of the current DOM hierarchy, so they are no longer part of the focus loop:
 
 {{"demo": "PortalTrapFocus.js"}}
+
+### Using a toggle inside the trap
+
+The most common use case for the `TrapFocus` component is to maintain focus within a [`Modal`](/base/react-modal/) component that is entirely separate from the element that opens the modal.
```

Comment:
This should be either "modal" or `ModalUnstyled`. I would go with the former since it refers to the generic concept of the component, and the user might not necessarily use the MUI Base `ModalUnstyled`.

```suggestion
The most common use case for the `TrapFocus` component is to maintain focus within a [modal](/base/react-modal/) component that is entirely separate from the element that opens the modal.
```

Reply:
Ah, great point! I was mixing up `@mui/base/UnstyledModal` with `@mui/material/Modal`

I'll go with the recommendation then!